idb/sig_type.go

```diff
@@ -0,0 +1,42 @@
+package idb
+
+import (
+	sdk_types "github.com/algorand/go-algorand-sdk/types"
+)
+
+type SigType string
+
+const (
+	Sig  SigType = "sig"
+	Msig SigType = "msig"
+	Lsig SigType = "lsig"
+)
+
+func IsSigTypeValid(sigtype SigType) bool {
+	return (sigtype == Sig) || (sigtype == Msig) || (sigtype == Lsig)
```

Comment:
The lookup map is faster and the SigTypeEnumString means 1 fewer thing to remember when we add a new signature. Could you add it back?

Reply:
done